docs/faq.jade

```diff
@@ -125,7 +125,7 @@ block content
 
     <hr id="arrow-functions" />
 
-    **Q**. I'm using an arrow function for a [virtual](./guide.html#virtuals), getter/setter, or [method](./guide.html#methods) and the value of `this` is wrong.
+    **Q**. I'm using an arrow function for a [virtual](./guide.html#virtuals), middleware (hooks), getter/setter, or [method](./guide.html#methods) and the value of `this` is wrong.
```

Comment:
Would be good to link to the [middleware docs](http://mongoosejs.com/docs/middleware.html) here